### PR TITLE
Adding jscs

### PR DESCRIPTION
**Pull request by elliotchance**
[Review on Reviewable](https://reviewable.io/reviews/glomotion/tesseract-js/8)

### CODE REVIEW

**Comment by elliotchance** [comment]
Do not merge this yet.

File: .jscsrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"preset": "google"
```
Comment (elliotchance) [inline .jscsrc]: I picked google, but there are many others to choose from:

http://jscs.info/overview.html#presets

**Comment by elliotchance** [comment]
This relates to #7 and #4

**Comment by glomotion** [comment]
Nice! Looks good. let me know when you want me to merge.

**Comment by elliotchance** [comment]
It can be merged as soon as the build is green. There are lots of little code style changes to be made to make it pass, see the build:

https://travis-ci.org/glomotion/tesseract-js/jobs/57892920

**Pull request closed without merging**